smoke_test.go

```diff
@@ -99,7 +149,17 @@ func initializeTestDatabaseConnection(t *testing.T) (conn *sql.Conn, cleanUpFunc
 
 	ctx := context.Background()
 
-	db, err := sql.Open(DoltDriverName, "file://"+dir+"?commitname=Billy%20Batson&commitemail=shazam@gmail.com&database=testdb&multistatements=true")
+	query := url.Values{
+		"commitname":      []string{"Billy Batson"},
+		"commitemail":     []string{"shazam@gmail.com"},
+		"database":        []string{"testdb"},
+		"multistatements": []string{"true"},
+	}
+	if clientFoundRows {
+		query["clientfoundrows"] = []string{"true"}
+	}
+	dsn := url.URL{Scheme: "file", Path: dir, RawQuery: query.Encode()}
```

Comment:
Very much appreciate how you're just openly curb-stomping @bheni with this change, you love to see it.